[WIP] Update Install Guide for v4.0.0 (#169)
libremfg/rhize-docs: content/deploy/install/keycloak.md

The other services do not need authorization but do need client authentication.
By default you need to add only the client ID.

For example, to create the Workflow client:
1. In the side menu, select **Clients > create client**.
1. For **Client ID**, enter `{{< param application_name >}}Workflow`
1. Configure the **Capability config**:
    - **Client Authentication**: On
1. Select **Next**, then **Save**.

> Repeat the process above for each of the following services with the given alterations:

Reviewer: ```suggestion
Repeat the preceding process for each of the following services with the given alterations:
```

Reviewer: > don't use directional language to orient the reader[^1]

[^1]: https://developers.google.com/style/accessibility

    - **Display on consent screen**: `On`
    - **Include in token scope**: `On`
1. **Create**.

#### Create audience mappers
Select the **Mappers** tab, then **Configure new mapper**. Add an audience mapper for the DB client:
   - **Mapper Type**: `Audience`
   - **Name**: `{{< param db >}}AudienceMapper`
   - **Include Client Audience**: `{{< param db >}}`
   - **Add to ID Token**: `On`
   - **Add to access token**: `On`

> Repeat the process above for each of the following services with the given alterations:

Reviewer: ```suggestion
Repeat the process preceding for each of the following services with the corresponding values in the table.
```

1. Select `{{< param application_name >}}ClientScope` from the list.
1. **Add > Default**.

> Repeat the process above for each of the following services:
> 
> |                   Name                   |
> | :--------------------------------------: |
> |               `dashboard`                |
> |  `{{< param application_name >}}Agent`   |
> |  `{{< param application_name >}}ISA95`   |
> |   `{{< param application_name >}}KPI`    |
> |    `{{< param application_name >}}UI`    |
> | `{{< param application_name >}}Workflow` |
> 

Reviewer: ```suggestion
Repeat the preceding process for each of the following services:

-  `dashboard`               
-  `{{< param application_name >}}Agent`
-  `{{< param application_name >}}ISA95` 
-   `{{< param application_name >}}KPI` 
-   `{{< param application_name >}}UI`  
- `{{< param application_name >}}Workflow` 

```

1. For **Temporary**, choose `Off`.
1. **Save**.

> Repeat the process above for each of the following services with the given alterations:

Reviewer: ```suggestion
Repeat the preceding process for each of the following services with the given alterations:
```
